sympy/concrete/summations.py

```diff
@@ -483,14 +483,13 @@ def is_convergent(self):
             return S.false
 
         ### ------------- integral test -------------- ###
-        if is_decreasing(sequence_term, interval):
```

Comment:
I presume this test was there because it is easier to answer this question that it is to do the integral (in general). If that is the case, I would prefer a modified or additional test that gets at the heart of what is being attempted. It appears that the problem in this case is that even though the function is decreasing in the range of interest, it is not doing so in a monotonic way -- there is an inflection at exp(7/12).